urbit/lib/eth-event-viewer.hoon

```diff
@@ -0,0 +1,33 @@
+/-  *eth-event-viewer
+=,  ethereum-types
+|%
+::
+++  json-to-view-action
```

Comment:
Same as previous comment applies here, except this would go in `++  dejs  |%`.

Reply:
:+1: